isbn/suche.html

```diff
@@ -141,7 +141,16 @@
             if (render(currentRecord.produktSigel) == '') {
                 $('#'+currentRecord.id).append("<td>"+render(currentRecord.umfang)+"</td>");
             } else {
-                $('#'+currentRecord.id).append("<td>"+render(currentRecord.umfang)+' ['+render(currentRecord.produktSigel)+"]</td>");
+                var siegelMitZDBAlsLinks = "";
+                for (var i=0; i<currentRecord.produktSigel.length; i++) { 
+                    if (currentRecord.produktSigel[i].startsWith("ZDB-")){ // Siegel die mit ZDB- beginnen einzeln als Links darstellen
+                        siegelMitZDBAlsLinks += '<a href="http://dispatch.opac.dnb.de/DB=1.2/CMD?ACT=SRCHA&IKT=8521&SRT=LST_os&TRM='+ currentRecord.produktSigel[i] + '" '  + 'target="_blank">' + currentRecord.produktSigel[i] + '</a> ';
+                    }
+                    else{ // andere Siegel normal darstellen
+                        siegelMitZDBAlsLinks += currentRecord.produktSigel[i] + " "; //Trenner 
+                    }
+                }
+                $('#'+currentRecord.id).append("<td>"+render(currentRecord.umfang)+' ['+ siegelMitZDBAlsLinks + "]</td>");                
```

Comment:
Konsistenterweise sollte man die Formatierung in die `rendering.js` packen als eine neue Funktion `renderSigel`.